visualization/dashboard/src/MLIDashboard/Controls/WhatIfTab/WhatIfTab.tsx

```diff
@@ -1254,13 +1291,27 @@ export class WhatIfTab extends React.PureComponent<IWhatIfTabProps, IWhatIfTabSt
     }
 
     private setSelectedCohort(event: React.FormEvent<HTMLDivElement>, item: IDropdownOption): void {
+        this.buildRowOptions(item.key as number);
         this.setState({
             selectedCohortIndex: item.key as number,
             selectedPointsIndexes: [],
             showSelectionWarning: false,
         });
     }
 
+    private buildRowOptions(cohortIndex: number): void {
+        this.props.cohorts[cohortIndex].sort(JointDataset.IndexLabel);
+        this.rowOptions = this.props.cohorts[cohortIndex]
+            .unwrap(JointDataset.IndexLabel)
+            .map((index) => {
+                return {
+                    key: index,
+                    text: localization.formatString(localization.WhatIfTab.rowLabel, index.toString()) as string,
```

Comment:
weird that a string needs to be casted to a string here...

Reply:
oh, thats the localization tool. The output of formatString is either a string (the only way I ever use it) or an array of strings. Not sure why they'd want a single function for what seems two different cases... so we cast.